crypto/ct/ct_sct_ctx.c

```diff
@@ -230,10 +230,10 @@ int SCT_CTX_set1_cert(SCT_CTX *sctx, X509 *cert, X509 *presigner)
     return 0;
 }
 
-static int ct_public_key_hash(X509_PUBKEY *pkey, unsigned char **hash,
-                              size_t *hash_len)
+__owur static int ct_public_key_hash(X509_PUBKEY *pkey, unsigned char **hash,
```

Comment:
Many X509 functions aren't constified, so we need a non-const `X509_PUBKEY` that we can pass to them. These parameters absolutely should be made const if/when the X509 functions are corrected.